app/controllers/sign_in/service_account_application_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module SignIn
+  class ServiceAccountApplicationController < ActionController::API
+    include SignIn::Authentication
```

Comment:
Are we using anything from `SignIn::Authentication` anymore? We could theoretically remove this `include`, and then you wouldn't have to `skip_before_action :authenticate`